File: Python/Gadget.py
class Gadget:
    def __init__(self, id, nama, harga, stok, gambar):
        self.__id = id
        self.__nama = nama
        self.__harga = harga
        self.__stok = stok
        self.__gambar = gambar

    # Getter & Setter
    def get_id(self): return self.__id
    def set_id(self, id): self.__id = id

    def get_nama(self): return self.__nama
    def set_nama(self, nama): self.__nama = nama

    def get_harga(self): return self.__harga
    def set_harga(self, harga): self.__harga = harga

    def get_stok(self): return self.__stok
    def set_stok(self, stok): self.__stok = stok

    def get_gambar(self): return self.__gambar
    def set_gambar(self, gambar): self.__gambar = gambar

    def tampilkan(self):
        print(f"ID     : {self.__id}")
        print(f"Nama   : {self.__nama}")
        print(f"Harga  : {self.__harga}")
        print(f"Stok   : {self.__stok}")
        print(f"Gambar : {self.__gambar}")

File: PHP/Gadget.php
<?php
class Gadget {
    private $id;
    private $nama;
    private $harga;
    private $stok;
    private $gambar;

    public function __construct($id, $nama, $harga, $stok, $gambar) {
        $this->id = $id;
        $this->nama = $nama;
        $this->harga = $harga;
        $this->stok = $stok;
        $this->gambar = $gambar;
    }

    // Getter & Setter
    public function getId() { return $this->id; }
    public function setId($id) { $this->id = $id; }

    public function getNama() { return $this->nama; }
    public function setNama($nama) { $this->nama = $nama; }

    public function getHarga() { return $this->harga; }
    public function setHarga($harga) { $this->harga = $harga; }

    public function getStok() { return $this->stok; }
    public function setStok($stok) { $this->stok = $stok; }

    public function getGambar() { return $this->gambar; }
    public function setGambar($gambar) { $this->gambar = $gambar; }

    // Tampilkan dalam tabel HTML
    public function tampilkan() {
        echo "<tr>
                <td>{$this->id}</td>
                <td>{$this->nama}</td>
                <td>{$this->harga}</td>
                <td>{$this->stok}</td>
                <td><img src='{$this->gambar}' width='80'></td>
                <td>
                    <a href='?aksi=update&id={$this->id}'>Update</a> | 
                    <a href='?aksi=hapus&id={$this->id}'>Hapus</a>
                </td>
              </tr>";
    }
}
?>
